project_redss/lib/dataset_specification.py

```diff
@@ -0,0 +1,57 @@
+# Note: This module project-specific and not yet suitable for migration to Core
+from core_data_modules.cleaners import somali
+
+from project_redss.redss_code_translators import *
+
+
+class CodingPlan(object):
+    def __init__(self, raw_field, coded_field, coda_name, cleaner=None, code_translator=None,
+                 time_field=None):
+        self.raw_field = raw_field
+        self.coded_field = coded_field
+        self.coda_name = coda_name
```

Comment:
Is coda_name the dataset_name? or some other name?

Reply:
Used to be a few things. Will change now that's only filename